src/Styles/Player.scss

```diff
@@ -0,0 +1,13 @@
+@import './_variables';
+@import './_mixins';
+
+.minus-btn,
+.plus-btn {
+  @include navy-button();
```

Comment:
These buttons looks really good. Maybe we can make them a little bit shorter so that they are squares.